From 4488170af944163b62051b2afee60386a9973c72 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Thu, 31 Mar 2016 10:23:48 +0200
Subject: [PATCH] remove building of provisioning service

The provisioning service is not used in this build and depends on boost_thread
which would add an additional unused dependency.

Change-Id: Ic7edab3ae96d77339ea0f48fc76ab2534c49b6ba
Signed-off-by: Hauke Mehrtens <hauke.mehrtens@intel.com>
---
 resource/src/SConscript | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/resource/src/SConscript
+++ b/resource/src/SConscript
@@ -116,5 +116,5 @@ oclib_env.UserInstallTargetHeader(header
 oclib_env.UserInstallTargetHeader(header_dir + 'CAManager.h', 'resource', 'CAManager.h')
 
 # Add Provisioning library
-if target_os in ['linux', 'android', 'tizen'] and env.get('SECURED') == '1':
-        SConscript('../provisioning/SConscript')
+# if target_os in ['linux', 'android', 'tizen'] and env.get('SECURED') == '1':
+#         SConscript('../provisioning/SConscript')
